python/lsst/ts/mthexapod/hexapod_csc.py

```diff
@@ -547,6 +556,7 @@ async def do_move(self, data):
         as it allows running with invalid compensation coefficients or inputs.
         """
         self.assert_enabled()
+        self.move_event.set()
```

Comment:
Should we set the event after knowing the low-level controller accepts this command? Under `self._move()`?

Reply:
This event merely indicates that a move command has been received. I'll rename it to add clarity.